docs/sources/explore/_index.md

```diff
@@ -44,12 +44,14 @@ To access Explore:
 
    {{< figure src="/static/img/docs/explore/panel_dropdown-7-4.png" class="docs-image--no-shadow" max-width= "650px" caption="Screenshot of the new Explore option in the panel menu" >}}
 
-1. Choose your data source from the dropdown in the top left. [Prometheus](https://grafana.com/oss/prometheus/) has a custom Explore implementation, the other data sources use their standard query editor.
-1. In the query field, write your query to explore your data. There are three buttons beside the query field, a clear button (X), an add query button (+) and the remove query button (-). Just like the normal query editor, you can add and remove multiple queries.
```

Comment:
The second bullet point is confusing, we don't even have these elements in the UI. The experience of writing a query in Explore is very much like in other places so I think we could simply cross-link with other docs.

Reply:
In the case of the steps, they should all be numbered `1.` our MD auto-enumerates. Also, I think starting with `2.` might confuse future contributors

Reply:
Fixed in 26a300de5244a56147f23f3a551a5297dbc29d36